compose/install.md

```diff
@@ -97,6 +97,8 @@ Docker Compose. To do so, follow these steps:
 <div id="linux" class="tab-pane fade" markdown="1">
 ### Install Compose on Linux systems
 
+*[The package repository set up for Docker instalation](https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#upgrade-docker-ce) contains a docker-compose package. If you installed Docker this way, simply run* `sudo apt-get install docker-compose`.
```

Comment:
nit typo: "instalation"

@shin- PTAL and make sure this is accurate -- do we have `docker-compose` in our official CE repos?

Reply:
To my knowledge, we don't - and the version in the default distro packages tend to be severely outdated. @andrewhsu can confirm.